src/api/getUserAndToken.ts

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+
+function parseJwt(token: any) {
+  try {
+    const base64Payload = token.split('.')[1]; // Get the payload part
+    const payload = Buffer.from(base64Payload, 'base64').toString('ascii');
+    return JSON.parse(payload);
+  } catch (e) {
+    console.error('Error decoding token:', e);
+    return null;
+  }
+}
+
+const be_url = process.env.REACT_APP_BE_URL || '';
+
+const getJwtTokenFromLocalStorage = () => {
+  const token = localStorage.getItem('token');
+  return token;
+};
+
+export const fetchUserAndToken = async () => {
+  try {
```

Comment:
Dont use this , 
use the one i created in utils , no need to cache token now will do that later